radarr/sqp/sqp-1-2160p.yml

```diff
@@ -1,16 +1,72 @@
-# Updated: 2023-08-02
+###################################################################################################
+# Recyclarr Configuration Template: SQP-1 (2160p)                                                 #
+# Updated: 2023-08-13                                                                             #
```

Comment:
Just a thought but I bet we could come up with a script that automatically adds this header *and* updates the date stamp for files that are modified in Git. If you are interested in that, I can try to come up with something. Shouldn't be too hard.

Reply:
Cool - I'll resolve here and open an issue for us to talk it over. Can take this as a leisurely one to implement!